src/utils.js

```diff
@@ -90,23 +90,13 @@ export const specialFilters = {
   }
 };
 
-export function select (...fields) {
-  return result => _.pick(result, ...fields);
-}
-
-export function selectMany (...fields) {
-  const selector = select(...fields);
-
-  return function (result) {
-    if (Array.isArray(result)) {
-      return result.map(selector);
+export function select (fields) {
+  return result => {
+    if (!(fields && Array.isArray(fields))) {
```

Comment:
Maybe I'm just tired but what is this doing? I'm having  tough time following the logic.

Can we switch it to:

```js
if (fields && Array.isArray(fields)) {
  return _.pick(result, ...fields);
}

return result;
```